keyboards/meira/meira.h

```diff
@@ -54,6 +54,7 @@ void reset_keyboard_kb(void);
 
 #define LAYOUT_ortho_4x12 KEYMAP
 #define KC_LAYOUT_ortho_4x12 KC_KEYMAP
+#define LAYOUT_kc_ortho_4x12 KC_KEYMAP
```

Comment:
These are redundant.

Reply:
Yes, but for some reason, this change switched them in some of the community layouts from KC_LAYOUT_ortho_4x12 to LAYOUT_kc_ortho_4x12 https://github.com/qmk/qmk_firmware/commit/4cfd1e30fc320dee84840b7af9ed474a6afcfaa6 

Other places still have KC_LAYOUT_ortho_4x12 defined, so I left it to avoid more things breaking.

Reply:
@drashna Is this waiting on me to change this before merging? If so, which is the preferred format? Removing one will also likely require changes elsewhere.

Reply:
Sorry, this is fine - we can address this later!